environments/dev/terraform/variables.tf

```diff
@@ -1,9 +1,3 @@
-variable kubeconfig_dir_path {
-  type = string
-  description = "Local path where all kubeconfigs generated in the environment should be written."
-}
-
 locals {
-  processing_kubeconfig_dir = "${var.kubeconfig_dir_path}/processing"
   vault_prefix = "secret/dsde/monster/dev"
```

Comment:
Can this variable be referenced from the common script?
I noticed you had to specify the vault path there, too.

Reply:
I'd like to wait until we instantiate our prod environment to see where it makes sense to parameterize this. I'm sure there are ways we could do it, but I'm not sure which makes the most sense right now.